lib/pg_party/hacks/postgresql_database_tasks.rb

```diff
@@ -8,18 +8,14 @@ def run_cmd(cmd, args, action)
           return super
         end
 
-        partitions = begin
-          ActiveRecord::Base.connection.select_values(<<-SQL)
-            SELECT
-              inhrelid::regclass::text
-            FROM
-              pg_inherits
-            JOIN pg_class AS p ON inhparent = p.oid
-            WHERE p.relkind = 'p'
-          SQL
-        rescue
-          []
-        end
+        partitions = ActiveRecord::Base.connection.select_values(<<-SQL)
+          SELECT
+            inhrelid::regclass::text
+          FROM
+            pg_inherits
+          JOIN pg_class AS p ON inhparent = p.oid
+          WHERE p.relkind = 'p'
+        SQL
```

Comment:
@igor-alexandrov I fixed the failing test by removing it 😆 

I don't think it ever really made sense to rescue an error here anyways

Reply:
Yeaahhh! I love this approach, dude!